### PR TITLE
updates for the Ip price increase

### PR DESCRIPTION
**Pull request by ksigler7**

### CODE REVIEW

**Comment by ksigler7** [comment]
@Whatthefoxsays Can you go ahead and review this when you get the chance, but not merge it in? We want it to be ready to merge first thing in the morning.

File: source/Classroom/Basics/Account/adding_an_additional_dedicated_ip_to_your_account.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ We do not recommend adding a new IP to automated warmup if you have no other IP
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Account/adding_an_additional_dedicated_ip_to_your_account.md]: On line 13, where it says "click the "Add a Dedicated IP" button

Change to:
Click **Add a Dedicated IP**.

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 2]
```diff
@@ -1,86 +1,69 @@
...
   keywords: invoice, bill, receipt, refund, line, item, marketing, campaigns, charge, contacts, usage, overage
-title: How to read your invoice
+title: Reading your invoice
 weight: 0
 layout: page
 zendesk_id: 
 navigation:
   show: true
 ---
-When you receive your invoice on the 1st of each month, you might have questions. This article aims to clarify what your invoices mean, and to help you learn more about what you are paying for.
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: I don't know that I would cast the invoice in a negative light right off the bat. Maybe something like:

"Your SendGrid invoice contains a lot of information, which may be difficult to read. Here's information on how to read your invoice:"

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 3]
```diff
@@ -1,86 +1,69 @@
...
 
  
 
-# 2. Billing Email
+{% anchor h2 %}
+2. Billing Email
+{% endanchor %}
 
-**The email address that your invoices are sent to.** This address defaults as the primary email address on file for your account, but can be updated to a different address from the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you override the billing email address in this way, the primary email address on file will stop receiving invoices. 
+**The email address where you receive SendGrid statements.** By default, this is the primary email address on file in your account, but you can update it to a different address from the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you override the billing email address in this way, the primary email address on file will stop receiving invoices.
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: By default, this is the primary email address on file in your account. You can update it at any time from the [Plan & Billing Details] page.

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 4]
```diff
@@ -1,86 +1,69 @@
...
 
- 
+![]({{root_url}}/images/invoice_legend.png)
 
-# 1. Physical Address
+{% anchor h2 %}
+1. Physical Address
+{% endanchor %}
 
 **The physical address of your business.** This address is the same address you entered when you first signed up. If you would like to update the address as it appears on your invoices, you can click on the "Change Invoice Address" button on the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you need to change the address on an existing invoice, please send an email to [billing@sendgrid.com](mailto:billing@sendgrid.com).
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: If you would like to update the address as it appears on your invoices, click **Change Invoice Address** on the [Plan & Billing Details] page.

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 5]
```diff
@@ -1,86 +1,69 @@
...
-**Newsletter Usage** - This item refers to the total usage charge for sending during the previous month with Legacy Newsletter features via either the API or the UI.
-
-Please refer to "Newsletter Usage" in the Usage Summary section below for more information on how this charge is calculated. 
-
- 
-![]({{root_url}}/images/invoice_pro100k_1.png)
-
-*This item refers to any charge for the previous month for the base plan price. Under normal circumstances, this item will always show a charge of $0.00 for this service period.
-
+This describes which plan you are on and describes the total usage charge for sending during the previous month. In this case, the user switched from the Essentials plan to the Pro plan in the middle of the previous month, and the charge reflects a credit for the part of the month they weren't on Essentials and a charge for the part of the month where they switched to Pro.
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: This describes the plan you subscribe to and the total charge for usage in the previous month.

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 6]
```diff
@@ -1,86 +1,69 @@
...
 
 ![]({{root_url}}/images/invoice_IP_Address.png)
 
-**IP Address** - This charge indicates you have an additional IP Address(es) allocated to your account. Specifically, this refers to any IP Address(es) beyond the first, which is always included in the base plan charge.
-
-Additional IPs cost $20.00 per IP/per month for Pro 100k plans and above. This example charge is for a single additional IP address for a single month. 
+**IP Address** - These charges indicates you have additional IP Addresses allocated to your account beyond the first, which is included in the base plan charge. There are two lines because one IP Address was added before the IP Addresses price increased, and other was added after.
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: These charges indicate that you have additional IP Addresses allocated to your account.

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 7]
```diff
@@ -1,86 +1,69 @@
...
-  
+Refer to "Contact List Usage" in the Usage Summary section below for more information on how this charge is calculated. 
 
-# 4. Usage Summary
+{% anchor h2 %}
+4. Usage Summary
+{% endanchor %}
 
-This section breaks down how certain charges from the above section are calculated. 
+*This is how charges from the Usage Summary section are calculated:*
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: Remove *

File: source/Classroom/Basics/Billing/how_to_read_your_invoice.md [hunk 8]
```diff
@@ -97,24 +80,20 @@ The overage charge rate for the Pro 100k plan is $0.00085 per extra email(20,000
...
 
 ![]({{root_url}}/images/invoice_payment_summary.png)
 
-This area shows the payment type, and last 4 digits of the card that was charged. For more information about updating your payment method, [click here](https://sendgrid.com/docs/Classroom/Basics/Billing/update_your_credit_card_and_resubmit_payments.html). 
-
-
- 
+The Transaction Summary shows the total charges for the month, and the Payment Summary area shows the payment type, and last 4 digits of the card that was charged. For more information about updating your payment method, [click here](https://sendgrid.com/docs/Classroom/Basics/Billing/update_your_credit_card_and_resubmit_payments.html). 
 
 ***These line items will always reflect the plan you currently have, not necessarily "Pro 100k", which is shown for the purposes of demonstration.**
```
Comment (Whatthefoxsays) [inline source/Classroom/Basics/Billing/how_to_read_your_invoice.md]: Remove extra * and make this a note.

**Pull request merged**